Source/MSBuildTools.Unity/Packages/com.microsoft.msbuildforunity/Editor/ProjectGenerator/MSBuildTemplates/MSBuildForUnity.Common.props.template

```diff
@@ -1,7 +1,11 @@
 <Project>
```

Comment:
Does this file go into the NuGet package or get distributed as part of the csproj?
Having the requirement be on the side of the NuGet package to determine where it's put feels like it's less customizable for somebody using MSBuildForUnity with arbitrary NuGet feeds.

Reply:
This file is generated by MSBuildForUnity

Reply:
Sorry, this immediate file is the template used to generate the common props file.